### PR TITLE
Added workaround mentioned in the bug -- Javascript version :)

### PR DESCRIPTION
**Pull request by Jayasankar-m**
Added 2 files incorporating the workaround - MakeSequential in javascript

### CODE REVIEW

**Review by Jayasankar-m — COMMENTED** [review]
point to the spec file with the workaround

File: test/async/spec.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    }
+}
+
+describe('[async] angularjs homepage todo list', function() {
+  it('should add a todo', async function() {
+    await browser.get('http://localhost:3000/index.html');
+    await element(by.model('todoList.todoText')).sendKeys('write first protractor test');
+    await element(by.css('[value="add"]')).click();
+    await element.all(by.repeater('todo in todoList.todos'))
+      .filter(MakeSequential(function(todo) {return todo.getText().then(label =>{console.log(label); return label.indexOf('#10') !== -1})}))
```
Comment (dubzzz) [inline test/async/spec.js]: Would be great if you remove the console.log and switch to arrow function whenever possible

**Review by dubzzz — CHANGES_REQUESTED** [review]
Thanks a lot for this solution

I will merge it as soon as you provide a version with no logs and arrow functions (optional)

**Review by Jayasankar-m — COMMENTED** [review]
Removed console.log and added arrow functions as suggested

**Comment by dubzzz** [comment]
Thanks a lot for the PR

**Pull request merged**